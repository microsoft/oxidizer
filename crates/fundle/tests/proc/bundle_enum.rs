
#[fundle::bundle]
enum Foo {}

fn main() {}
