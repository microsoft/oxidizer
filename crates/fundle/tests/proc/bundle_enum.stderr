error: fundle::bundle can only be applied to structs
 --> tests/proc/bundle_enum.rs:3:1
  |
3 | enum Foo {}
  | ^^^^^^^^^^^
